### DISCUSSION TITLE
Restarting Milvus and unable to find data, what is the situation?

### DISCUSSION THREAD

**Discussion by lisong888** [milvus-io/milvus#23518] [Q&A and General discussion] [answered]
Before restarting Milvus, the data in the library can be queried, but after restarting, the data cannot be queried. What is the situation? But the amount of data that can be seen in ATTU is that data cannot be queried.
![QQ图片20230419102843](https://user-images.githubusercontent.com/45256453/232956871-eb79723e-1e57-4c84-bc31-6f8a547e237c.png)

**Comment by yhmo** [comment]
What error is returned when you perform a search?

**Reply by lisong888** [reply, marked as answer]
Problem solved. The issue occurred in version 2.0, and I am currently using version 2.2.5 and will no longer encounter this issue
